test/ctia/test_helpers/aggregate.clj

```diff
@@ -50,17 +50,29 @@
                     (vector? (first values)) flatten-list-values)]
     (map string/lower-case flattened)))
 
+(defn- check-from-to
+  [from to]
```

Comment:
Suppose that both `from` and `to` are empty strings. Then this test would happily pass:

```clj
(<= (time/in-years
     (time/interval (tc/from-string "")
                    (tc/from-string "")))
    1) ;; => true
```

I tend to be wary of possibly empty strings in date helpers, as multiple Clojure date libs are known to be a bit permissive there.

More generally, in test helpers (or around assertions) I try to add extra checks for preventing false positives, negatives etc.